plugins/svn/README.md

```diff
@@ -0,0 +1,10 @@
+## svn
+
+**Maintainer:** 
+
+This plugin adds some useful functions.
+
+### Usage
+
+See the [wiki](https://github.com/robbyrussell/oh-my-zsh/wiki/Plugin:svn) for a list of functions provided by the plugin.
```

Comment:
When you integrate the how-to use it with agnoster, add it after the `Functions` section, inside a `Usage` section.